SBBDesignSystemMobileSwiftUI/Views/SBBUpDnCounterView.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  Copyright © 2023 SBB. All rights reserved.
+//
+
+import SwiftUI
+
+public struct SBBUpDnCounterView: View {
```

Comment:
For what stand this name? I do not get the "Up Dn" counter abbreviation...

Reply:
This is the DSM Name for Stepper: https://www.figma.com/file/WOtLIam1xwrqcgnAITsEhV/Design-System-Mobile?type=design&node-id=372%3A11403&t=xNqeiEa4K00eOoTv-1